cmd/earth/main.go

```diff
@@ -163,6 +166,13 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 			Usage:       "Do not use cache while building",
 			Destination: &app.noCache,
 		},
+		&cli.StringFlag{
+			Name:        "config",
+			Value:       "~/earthly/config.yaml",
```

Comment:
Should `~` be replaced by `os.Getenv("HOME")`?

Reply:
good catch!